packages/dashboard/src/components/csvDownloadButton/getDescribedTimeSeries.tsx

```diff
@@ -0,0 +1,33 @@
+import { DescribeTimeSeriesCommand, IoTSiteWiseClient } from '@aws-sdk/client-iotsitewise';
+
+export const getDescribedTimeSeries = async ({
+  client,
+  assetId,
+  propertyId,
+  alias,
+}: {
+  client: IoTSiteWiseClient;
+  assetId?: string;
+  propertyId?: string;
+  alias?: string;
+}) => {
+  const isModeledData = assetId && propertyId;
+
+  if (!isModeledData && !alias) {
+    return {};
```

Comment:
this situation shouldnt happen unless there is an issue with the data so i will throw an error here instead

Reply:
I think you can use this library Tracy introduced with his resource explorer change to guard against this - `import invariant from 'tiny-invariant';`

Reply:
oh yes! ty! had used this in the v1 but completely forgot